CONTRIBUTING.md

```diff
@@ -50,9 +71,17 @@ To contribute a patch, the workflow is as follows:
   1. Create topic branch
   1. Commit patches
 
+For GUI related issues or pull requests, the https://github.com/bitcoin-core/gui repository should be used.
+For all other issues and pull requests, the https://github.com/bitcoin/bitcoin main repository should be used.
+
+For global refactoring or other global changes that touche GUI code, as well as
+non-GUI code, the main repository should be used.
```

Comment:
So if a PR only adds a new GUI feature, but needs to touch a couple of lines in other areas for that, does it go into the main repo or the gui one? (and would drahtbot be able to detect conflicts if it goes to the gui one?)

Reply:
I will run DrahtBot on the gui repo when it is created. Given that the GUI speaks with the node over interfaces, there shouldn't be too many inter-repo conflicts.

Reply:
Clarified, hopefully.